aries-site/src/examples/components/skeleton/DashBoardSkeletonExample.js

```diff
@@ -0,0 +1,92 @@
+import React, { useContext, useEffect, useState } from 'react';
+import {
+  Box,
+  Button,
+  Card,
+  CardBody,
+  CardHeader,
+  CardFooter,
+  Grid,
+  Heading,
+  Page,
+  PageContent,
+  Paragraph,
+  ThemeContext,
+  ResponsiveContext,
+} from 'grommet';
+import { adjustPad } from '../../templates/Card/utils';
+import { activities } from '../card/data';
+
+const skeleton = { message: { start: 'Loading', end: 'Content Loaded' } };
+
+const columns = {
+  xsmall: ['auto'],
+  small: ['auto'],
+  medium: ['auto'],
+  large: ['auto', 'auto'],
+  xlarge: ['auto', 'auto', 'auto'],
+};
+
+export const DashBoardSkeletonExample = () => {
+  const theme = useContext(ThemeContext);
+  const breakpoint = useContext(ResponsiveContext);
+  const [loading, setLoading] = useState(true);
+  // needed for skeleton paragraph
+  const skeletonAlign = loading ? 'none' : 'start';
+  useEffect(() => {
+    setTimeout(() => setLoading(!loading), 3000);
+  }, [loading]);
+
+  const placeholders = [{}, {}, {}];
+  const items = loading ? placeholders : activities;
+
+  return (
+    // passing skeleton to Grid will skeletize all of the children that are
+    // text, button, box, paragraph, heading
+    <Page height="large">
```

Comment:
```suggestion
    <Page>
```
Instead of setting the height here, we can set it on the "Example" container and do a custom value (since it won't be exposed)